python/sglang/srt/server_args.py

```diff
@@ -2366,6 +2367,11 @@ def add_cli_args(parser: argparse.ArgumentParser):
             default=ServerArgs.hicache_storage_backend_extra_config,
             help="A dictionary in JSON string format containing extra configuration for the storage backend.",
         )
+        parser.add_argument(
+            "--hicache-storage-pass-prefix-keys",
```

Comment:
can we make it part of the "hicache-storage-backend-extra-config"?

Reply:
Sure, updated  @xiezhq-hermann